compute/cker/include/cker/train/operation/Pad.h

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright (c) 2024 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __NNFW_CKER_TRAIN_OPERATION_PAD_H__
+#define __NNFW_CKER_TRAIN_OPERATION_PAD_H__
+
+#include "cker/operation/Pad.h"
+
+namespace nnfw
+{
+namespace cker
+{
+namespace train
+{
+
+/*
+ * input_data will be transformed by PAD operation with padding options(such as constant C) to
+ * output_data
+ *
+ * input_data  ->  output_data
+ *   [0,1]     ->    [C,C,C,C]
+ *   [2,3]     ->    [C,0,1,C]
+ *             ->    [C,2,3,C]
+ *             ->    [C,C,C,C]
+ */
+template <typename T>
+inline void Pad(const int32_t *padding_data, int32_t pad_rank, const Shape &input_shape,
```

Comment:
My new patch overwrites @jyoungyun 's comment. I write here again instead of @jyoungyun .

https://github.com/Samsung/ONE/pull/12522#discussion_r1464521239

> IMHO, there is no need to re-define this function. We can use nnfw::cker::Pad directly in train backend. It makes unnecessary function call and we have to manage two set of code (Pad in train/operation and Pad in cpu/operation). Also it makes binary size bigger. (Of course, it's a very trivial issue. 😄)

Reply:
I agree with @jyoungyun . I'm going to use `nnfw::ckwer::Pad` in train backend.